swerve_point_generator/Coefs[] orient_coefs
swerve_point_generator/Coefs[] x_coefs
swerve_point_generator/Coefs[] y_coefs
int32[] spline_groups
float64[] wait_before_group
float64[] t_shift
bool[] flip
float64[] end_points
float64 initial_v
float64 final_v
---
talon_swerve_drive_controller/SwervePoint[] points 
float64 dt
